cluster/gce/gci/configure-helper.sh

```diff
@@ -706,9 +705,6 @@ function start-kube-addons {
   fi
   if [[ "${ENABLE_L7_LOADBALANCING:-}" == "glbc" ]]; then
     setup-addon-manifests "addons" "cluster-loadbalancing/glbc"
```

Comment:
oh, no i guess it can. it splits addon deployment, but consolidates loadbalancer deployment. So ill send a follow up.